ckanext/canada/tables/briefingt.yaml

```diff
@@ -128,12 +146,16 @@ resources:
       en: This field will display the date on which the Briefing Note was received in the addressee's office.
     obligation: Mandatory
     excel_required: true
+    form_required: true
     format_type: Date (Please format the data as YYYY-MM-DD)
     validation: This field must not be empty
     visible_to_public: Yes
     occurrence: Single
     datastore_type: date
     excel_column_width: 28
+    form_snippet: scheming/form_snippets/date.html
+    form_attrs:
+      style: 'width:155.97px'
```

Comment:
@wardi are you ok with width defined in pixels? I prefer percentage (if needed) so that it can adapt to screen size